app/views/memberevents/index.html.erb

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
This file still belongs to a controller called `memberevents` and should say `upcoming-events` if you renamed the controller.